annotations/.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Wed Jun 21 17:51:06 CEST 2023
```

Comment:
Please remove this whole `.gradle` directory from Git repository. It definitely shouldn't be added into VCS. This folder contains machine specific information and is used for caching on local ;)